interface/forms/misc_billing_options/new.php

```diff
@@ -34,110 +34,220 @@
+                        <div class="form-group clearfix">
+                                <label for='off_work_from' class="col-sm-3 form-inline"><?php echo xlt('Box 16. Date unable to work from');?>:</label>
+                                    <?php $off_work_from = $obj{"off_work_from"}; ?>
+                                        <input type="text" class='datepicker form-inline col-sm-2 ' name='off_work_from' id='off_work_from' value='<?php echo attr($off_work_from); ?>' title='<?php echo xla('yyyy-mm-dd'); ?>'>
+                                <label for='off_work_to'class="col-sm-3 form-inline"><?php echo xlt('Box 16. Date unable to work to');?>:</label>
+                                    <?php $off_work_to = $obj{"off_work_to"}; ?>
+                                        <input type="text"  class='datepicker col-sm-2' name='off_work_to' id='off_work_to' value='<?php echo attr($off_work_to); ?>' title='<?php echo xla('yyyy-mm-dd'); ?>'>
+                        </div>
+                        <div class="form-group">
+                            <label class="form-inline"><?php echo xlt('Box 17. Provider') ?>:
+                                <?php  genProviderSelect('provider_id', '-- ' . xl("Please Select") . ' --', $obj{"provider_id"}); ?>
+                            </label>
+                            <label class="form-inline"><?php  echo xlt('Box 17. Provider Qualifier'); ?>:
+                                <?php  echo generate_select_list('provider_qualifier_code', 'provider_qualifier_code', $obj{"provider_qualifier_code"}, 'Provider Qualifier Code');?>
+                            </label>
+                        </div>
+                        <div class="form-group clearfix">
+                            <label for='hospitalization_date_from' class="col-sm-3 form-inline"><?php echo xlt('Box 18. Hospitalization date from');?>:</label>
+                                <?php $hospitalization_date_from = $obj{"hospitalization_date_from"}; ?>
+                                    <input type="text" class='datepicker col-sm-2 ' name='hospitalization_date_from' id='hospitalization_date_from' value='<?php echo attr($hospitalization_date_from); ?>' title='<?php echo xla('yyyy-mm-dd'); ?>'>
+                            <label for='off_work_to'class="col-sm-3 form-inline"><?php echo xlt('Box 18. Hospitalization date to');?>:</label>
+                                <?php $hospitalization_date_to = $obj{"hospitalization_date_to"}; ?>
+                                    <input type="text"  class='datepicker col-sm-2'  name='hospitalization_date_to' id='hospitalization_date_to' value='<?php echo attr($hospitalization_date_to); ?>' title='<?php echo xla('yyyy-mm-dd'); ?>'>
+                        </div>
+                        <div class="form-group">
+                            <label><?php echo xlt('Box 20. Is Outside Lab used?'); ?>:
+                                <input type="checkbox" name="outside_lab" value="1"
```

Comment:
does this need an id of "outside_lab" ?

Reply:
Yes and no, yes for semantics, no as the input is wrapped in the label tag